tr_pud-ud-test.conllu

```diff
@@ -19212,8 +19212,8 @@
 1	Bu	bu	DET	DT	Definite=Def|Polarity=Pos	2	det	_	_
 2	olay	olay	NOUN	NN	Number=Sing	3	nmod:poss	_	_
 3	esnasında	esnas	NOUN	NN	Case=Loc|Number=Sing|Number[psor]=Sing|Person[psor]=3	4	obl	_	_
-4	gerçekleşen	gerçekleş	VERB	VB	Aspect=Perf|Mood=Ind|Number=Sing|Tense=Pres|VerbForm=Part	6	nmod:poss	_	_
-5	lerin	_	X	GW	Case=Gen|Number=Plur|Person=3	4	goeswith	_	_
+4	gerçekleşen	gerçekleş	VERB	VB	Aspect=Perf|Case=Gen|Mood=Ind|Number=Plur|Tense=Pres|Typo=Yes|VerbForm=Part	6	nmod:poss	_	_
+5	lerin	_	X	GW	_	4	goeswith	_	_
```

Comment:
This fix is uncontroversial, but it is actually odd for a PUD treebank to have `Typo=Yes` and/or `goeswith` at all. This is not a naturally occurring text. It has been translated into Turkish specifically for the purpose of UD annotation, so there shouldn't be any errors in the translation.

Reply:
This is a mistake during automatic segmentation during the annotation process. I do not think the original text would have this word split into two. Correct solution would be merging these tokens. I'd annotate as:
```
gerçekleşenlerin gerçekleş VERB    _ Case=Gen|Number=Plur|Tense=Pres|VerbForm=Part   5       acl     _       _
```

But it seems in PUD participles and verbal nouns are all split. (Also, `nomd(:poss)` is also the default for clausal modifiers that behave like nouns as the one above.)

Reply:
I didnt't realise that PUD translations are "error free". In these case I agree that we should retokenise it into _gerçekleşenlerin_. What deprel do you prefer,  `acl` or rather `nmod`?

Reply:
> I didnt't realise that PUD translations are "error free". In these case I agree that we should retokenise it into _gerçekleşenlerin_. What deprel do you prefer, `acl` or rather `nmod`?

From the global UD perspective I would say that if it is correctly tagged as `VERB`, then it heads a clause, meaning that it can be `acl` but not `nmod` or `nmod:poss`. But I haven't checked what's usually done in the Turkish treebanks.

Reply:
This is one of the tricky cases, and inconsistently annotated in different treebanks. These clauses behave like nouns (as in here, participating in a typical genitive - possessive construction). However, UD does not have dependency type for 'a nominal clause modifier of a noun'. Some treebanks use `acl`, and others use `nmod(:poss)`. I am more inclined for `acl` (the type of modification can be inferred from the morphological features), but I do understand the others wanting to treat these clauses like nouns.